src/main/java/com/gmail/goosius/siegewar/listeners/SiegeWarNationEventListener.java

```diff
@@ -243,19 +225,20 @@ public void onNationStatusScreen(NationStatusScreenEvent event) {
 	}
 	
 	/*
-	 * A nation being deleted with a siege means the siege ends.
+	 * A nation being deleted with a siege means the siege ends,
+	 * and a king may receive a refund.
 	 */
-	@EventHandler
-	public void onDeleteNation(DeleteNationEvent event) {
-		UUID kingUUID = event.getNationKing();
-		if (kingUUID == null)
-			return;
-		
-		Resident king = TownyUniverse.getInstance().getResident(kingUUID);
-		if (king != null)
-			SiegeWarMoneyUtil.makeNationRefundAvailable(king);
+	@EventHandler(priority=EventPriority.MONITOR, ignoreCancelled=true)
+	public void onDeleteNation(PreDeleteNationEvent event) {

```

Comment:
:question: 
- When will this method get triggered? 
   - If AFTER the king uses the nation delete command but BEFORE they confirm..... then wouldn't this method delete the siege even if the king cancels the delete?
   - If AFTER the king uses the nation delete command, and AFTER they confirm..... then wouldn't the given warning be too late to serve as a warning?

Reply:
It will be shown after the kings has used the delete command/used the `/n leave` command/after the upkeep kills the nation.

The warning is not being shown before the confirmation in the current SiegeWar. This may have been a regression that occured between Towny-with-SiegeWar and SiegeWar.

Reply:
The warning might be unneeded, since the player gets the warning and the nation refund available message at the same time.

Reply:
- Ah I see
- That's ok, but then on a separate topic the message text needs updating. It currently reads ```"WARNING!!!: If you delete your nation you will be refunded %s. This can be claimed later by using '/sw nation refund'"```
- Will me merge this PR now and update that lang string later ?
- Or perhaps will we update the lang string in this PR?. It would be```"For deleting the nation, you have been refunded %s. This can be claimed later by using '/sw nation refund'"```

Reply:
I'll update the strings with a second commit after double checking if we even need the warning message any more.

Reply:
Warning message is un-needed, remaining msg_siege_war_nation_refund_available is `A nation refund of %s is now available to you. It can be claimed using '/sw nation refund'` which I think covers the bases.

Reply:
Ah grand, I didn't realize they would get that